doc-surrealdb_versioned_docs/version-1.x/surrealql/statements/create.mdx

```diff
@@ -103,14 +103,47 @@ CREATE person:100 CONTENT {
 };
 ```
 
-## Clauses and Options
+## Options and clauses
+
+### Creating multiple records
+
+Multiple records or even multiple record types can be created by separating record names by commas.
+
+```surql
+--Note: meta::tb(id) returns just the table name portion of a record ID
+CREATE townsperson, cat, dog SET
+    created_at = time::now(),
+    name = "Just a " + meta::tb(id);
+
+--Output
+[
+    {
+        "created_at": "2024-03-19T03:12:05.079Z",
+        "id": "townsperson:p37ha2lngckp3v8tvf2j",
+        "name": "Just a townsperson"
+    },
+    {
+        "created_at": "2024-03-19T03:12:05.080Z",
+        "id": "cat:p1pwbjaq96nhhnuohjtc",
+        "name": "Just a cat"
+    },
+    {
+        "created_at": "2024-03-19T03:12:05.080Z",
+        "id": "dog:01vcxgdpuctdk354hzkp",
+        "name": "Just a dog"
+    }
+]
+```
```

Comment:
```suggestion
```surql
--Note: meta::tb(id) returns just the table name portion of a record ID
CREATE townsperson, cat, dog SET
    created_at = time::now(),
    name = "Just a " + meta::tb(id);
```
```bash title="Output"
[
    {
        "created_at": "2024-03-19T03:12:05.079Z",
        "id": "townsperson:p37ha2lngckp3v8tvf2j",
        "name": "Just a townsperson"
    },
    {
        "created_at": "2024-03-19T03:12:05.080Z",
        "id": "cat:p1pwbjaq96nhhnuohjtc",
        "name": "Just a cat"
    },
    {
        "created_at": "2024-03-19T03:12:05.080Z",
        "id": "dog:01vcxgdpuctdk354hzkp",
        "name": "Just a dog"
    }
]
```
```

Reply:
@Dhghomon I am unsure why GitHub highlights a suggestion but the suggestion is to remove the Output from the same copy block as the code. 

```
``` bash title="output"
[
    {
        "created_at": "2024-03-19T03:12:05.079Z",
        "id": "townsperson:p37ha2lngckp3v8tvf2j",
        "name": "Just a townsperson"
    },
    {
        "created_at": "2024-03-19T03:12:05.080Z",
        "id": "cat:p1pwbjaq96nhhnuohjtc",
        "name": "Just a cat"
    },
    {
        "created_at": "2024-03-19T03:12:05.080Z",
        "id": "dog:01vcxgdpuctdk354hzkp",
        "name": "Just a dog"
    }
]
```
```